src/CameraSensorUtil.hh

```diff
@@ -82,11 +82,12 @@ math::Matrix4d buildPerspectiveMatrix(
 ///            This value is negative if the plane is to be behind
 ///            the camera
 /// \return OpenGL projection matrix
-math::Matrix4d buildProjectionMatrix(double _imageWidth, double _imageHeight,
-                                     double _intrinsicsFx, double _intrinsicsFy,
-                                     double _intrinsicsCx, double _intrinsicsCy,
-                                     double _intrinsicsS, double _clipNear,
-                                     double _clipFar);
+math::Matrix4d GZ_SENSORS_VISIBLE buildProjectionMatrix(
```

Comment:
I think it would be better to compile `CameraSensorUtil.cc` with each of the [components](https://github.com/gazebosim/gz-sensors/blob/d44b7dcdfa37ed41f5fa075ef044072d52b7e5e2/src/CMakeLists.txt#L62) that need it rather than making it visible here since it now becomes part of our public ABI.

Reply:
testing alternative fix in [d5b64ef](https://github.com/gazebosim/gz-sensors/pull/392/commits/d5b64efa57665c0fab01964297b6af1a0f739aff)

Reply:
Windows CI looks good with the new changes.